src/Tasks/Microsoft.NET.Build.Tasks/GenerateBundle.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Build.Framework;
+using Microsoft.NET.HostModel.Bundle;
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.NET.Build.Tasks
+{
+    public class GenerateBundle : TaskBase
+    {
+        [Required]
+        public ITaskItem[] FilesToBundle { get; set; }
+        [Required]
+        public string AppHostName { get; set; }
+        [Required]
+        public bool IncludeSymbols { get; set; }
+        [Required]
+        public string OutputDir { get; set; }
+        [Required]
+        public bool ShowDisgnosticOutput { get; set; }
```

Comment:
```suggestion
        public bool ShowDiagnosticOutput { get; set; }
```

Just spotted a typo

Reply:
Thanks, fixed it in 77e4d367b557e788242cee0b3406b83b298b5b0a